.gitignore

```diff
@@ -30,6 +30,7 @@ faces-config.NavData
 *.lock.db
 .cache-main
 .cache-tests
+.jqwik-database
```

Comment:
:sunglasses: nice, I run into this from time to time

Reply:
Yeah thought about putting it in a separate PR, but just wanted to get this out the door (and backported)